colfer.go

```diff
@@ -147,6 +151,8 @@ type Struct struct {
 	Fields []*Field
 	// SchemaFile is the source filename.
 	SchemaFile string
+	// TagAdd is an optional source code addition.
+	TagAdd string
```

Comment:
If I wanted multiple tags on different lines, such as on Java/Python, wouldn't it be easier to make this a: TagAdd[] string instead?

Reply:
Sure. It requires a bit more thought on the input format though.

Why would you want multiple lines?

Reply:
Java annotations can become quite unwieldy, especially if qualifying them with package names too:

```
public class Test {
	@Deprecated @SuppressWarnings({"unchecked", "deprecation"}) @com.fasterxml.jackson.annotation.JsonProperty("x") @com.google.gson.annotations.SerializedName(value = "x", alternate = "X")
	String x;
}
```